[{"url":"https://api.github.com/repos/owncloud/core/issues/comments/30007193","html_url":"https://github.com/owncloud/core/issues/5925#issuecomment-30007193","issue_url":"https://api.github.com/repos/owncloud/core/issues/5925","id":30007193,"user":{"login":"karlitschek","id":867445,"avatar_url":"https://gravatar.com/avatar/87ce2b4531ee1a0c32b50e0d8f049224?d=https%3a%2f%2fidenticons.github.com%2f06e7098636caebfc9c30c9f52eb905df.png&r=x","gravatar_id":"87ce2b4531ee1a0c32b50e0d8f049224","url":"https://api.github.com/users/karlitschek","html_url":"https://github.com/karlitschek","followers_url":"https://api.github.com/users/karlitschek/followers","following_url":"https://api.github.com/users/karlitschek/following{/other_user}","gists_url":"https://api.github.com/users/karlitschek/gists{/gist_id}","starred_url":"https://api.github.com/users/karlitschek/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/karlitschek/subscriptions","organizations_url":"https://api.github.com/users/karlitschek/orgs","repos_url":"https://api.github.com/users/karlitschek/repos","events_url":"https://api.github.com/users/karlitschek/events{/privacy}","received_events_url":"https://api.github.com/users/karlitschek/received_events","type":"user","site_admin":false},"created_at":"2013-12-06t16:24:07z","updated_at":"2013-12-06t16:24:07z","body":"@blizzz what do you think? "},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/30017540","html_url":"https://github.com/owncloud/core/issues/5925#issuecomment-30017540","issue_url":"https://api.github.com/repos/owncloud/core/issues/5925","id":30017540,"user":{"login":"blizzz","id":2184312,"avatar_url":"https://gravatar.com/avatar/1898b25a9cb3aa1a0f0febd1359910b9?d=https%3a%2f%2fidenticons.github.com%2fca03acec6deadf508b75eb0cf9ba036c.png&r=x","gravatar_id":"1898b25a9cb3aa1a0f0febd1359910b9","url":"https://api.github.com/users/blizzz","html_url":"https://github.com/blizzz","followers_url":"https://api.github.com/users/blizzz/followers","following_url":"https://api.github.com/users/blizzz/following{/other_user}","gists_url":"https://api.github.com/users/blizzz/gists{/gist_id}","starred_url":"https://api.github.com/users/blizzz/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/blizzz/subscriptions","organizations_url":"https://api.github.com/users/blizzz/orgs","repos_url":"https://api.github.com/users/blizzz/repos","events_url":"https://api.github.com/users/blizzz/events{/privacy}","received_events_url":"https://api.github.com/users/blizzz/received_events","type":"user","site_admin":false},"created_at":"2013-12-06t18:34:20z","updated_at":"2013-12-06t18:34:20z","body":"i cannot reproduct it with\r\n\r\nmysql  ver 14.14 distrib 5.5.34, for debian-linux-gnu (x86_64) using readline 6.2\r\n\r\nand a recent owncloud. \r\n\r\n@catara could you retry with latest owncloud rc?"},{"url":"https://api.github.com/repos/owncloud/core/issues/comments/30180923","html_url":"https://github.com/owncloud/core/issues/5925#issuecomment-30180923","issue_url":"https://api.github.com/repos/owncloud/core/issues/5925","id":30180923,"user":{"login":"catara","id":1438420,"avatar_url":"https://gravatar.com/avatar/6e3f69c38798fd2a077d394a26fd8426?d=https%3a%2f%2fidenticons.github.com%2fefc9af9adcea8bbb6505ce691f9ee271.png&r=x","gravatar_id":"6e3f69c38798fd2a077d394a26fd8426","url":"https://api.github.com/users/catara","html_url":"https://github.com/catara","followers_url":"https://api.github.com/users/catara/followers","following_url":"https://api.github.com/users/catara/following{/other_user}","gists_url":"https://api.github.com/users/catara/gists{/gist_id}","starred_url":"https://api.github.com/users/catara/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/catara/subscriptions","organizations_url":"https://api.github.com/users/catara/orgs","repos_url":"https://api.github.com/users/catara/repos","events_url":"https://api.github.com/users/catara/events{/privacy}","received_events_url":"https://api.github.com/users/catara/received_events","type":"user","site_admin":false},"created_at":"2013-12-09t22:32:40z","updated_at":"2013-12-09t22:32:40z","body":"mysql problem gone now in:\r\n\r\n<b>owncloud 7.0 pre alpha (daily) build:2013-12-07t04:04:38+00:00</b>\r\nldap user and group backend0.4.1\r\napache/2.2.24 (unix) php/5.4.17 mod_ssl/2.2.24 openssl/0.9.8y dav/2 server at localhost port 80\r\nclient api version\tmysqlnd 5.0.10 - 20111026 - $id: e707c415db32080b3752b232487a435ee0372157\r\n<b>mysql-5.6.15-osx10.7-x86_64</b>\r\n\r\nnew problem now: could not determine base dn (maybe i'll open a new ticket)\r\n\r\n<i>(remember not to try to replicate it on a diff os, with a diff mysql... it would help users treating you seriously or with not wasting their time)</i>"}]